packages/apidom-ls/src/config/openapi/config.ts

```diff
@@ -83,4 +84,5 @@ export default {
   tag: tagMeta,
   xml: xmlMeta,
   schema: schemaMeta,
+  'path-template': pathTemplateMeta,
```

Comment:
Here we're attaching our meta (containing linting rules) to the `path-template` meta class.